src/components/AppLayout/Notifications/types.ts

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
```suggestion
import { ReactNode } from "react";
```

<img width="679" alt="Screenshot 2024-05-13 at 10 17 44 AM" src="https://github.com/deriv-com/ui/assets/93518187/99a79388-d92a-4a58-9aa9-b748fca6ffef">

Reply:
done